modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/application/management/v1/model/ApplicationResponseModel.java

```diff
@@ -443,29 +460,29 @@ public String toString() {
                 "    accessUrl: " + toIndentedString(accessUrl) + "\n" +
                 "    clientId: " + toIndentedString(clientId) + "\n" +
                 "    issuer: " + toIndentedString(issuer) + "\n" +
+                "    realm: " + toIndentedString(realm) + "\n" +
                 "    templateId: " + toIndentedString(templateId) + "\n" +
                 "    isManagementApp: " + toIndentedString(isManagementApp) + "\n" +
+                "    isB2BSelfServiceApp: " + toIndentedString(isB2BSelfServiceApp) + "\n" +
                 "    associatedRoles: " + toIndentedString(associatedRoles) + "\n" +
                 "    claimConfiguration: " + toIndentedString(claimConfiguration) + "\n" +
                 "    inboundProtocols: " + toIndentedString(inboundProtocols) + "\n" +
                 "    authenticationSequence: " + toIndentedString(authenticationSequence) + "\n" +
-                "    appRoleConfigurations: " + toIndentedString(appRoleConfigurations) + "\n" +
                 "    advancedConfigurations: " + toIndentedString(advancedConfigurations) + "\n" +
                 "    provisioningConfigurations: " + toIndentedString(provisioningConfigurations) + "\n" +
                 "    access: " + toIndentedString(access) + "\n" +
                 "}";
     }
 
     /**
-    * Convert the given object to string with each line indented by 4 spaces
-    * (except the first line).
-    */
+     * Convert the given object to string with each line indented by 4 spaces
+     * (except the first line).
+     */
     private String toIndentedString(Object o) {
 
         if (o == null) {
             return "null";
         }
-        return o.toString();
+        return o.toString().replace("\n", "\n");
```

Comment:
Why target is required to replace by the target itself ?

Reply:
The 4 spaces intending is not in this method scope. Shall we re-write the method comment ?

Reply:
Removed the unnecessary replace method by https://github.com/wso2/product-is/pull/20088
However this file is a copy of the ApplicationResponseModel.java[1] file in identity-api-server which is an auto generated class. Hence not changing the method comment. 


[1] - https://github.com/wso2/identity-api-server/blob/master/components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/gen/java/org/wso2/carbon/identity/api/server/application/management/v1/ApplicationResponseModel.java